apollo-router/src/configuration/migrations/0027.entity_cache_type_metrics.yaml

```diff
@@ -0,0 +1,5 @@
+description: Entity cache entity.type attribute renamed to graphql.type.name
```

Comment:
Should this file start with `0028-` instead of `0027.`?

Reply:
I was concerned this meant the migration wouldn't actually be run, but looks like all `*.yaml` files get run. So since it's just a cleanliness issue rather than a semantic issue, I fixed this in my PR #5990 (which adds another migration after this one).

Reply:
Yes, thanks for fixing this.